build-system/dep-check-config.js

```diff
@@ -139,6 +139,10 @@ exports.rules = [
           'src/service/variable-source.js',
       'extensions/amp-a4a/0.1/amp-a4a.js->' +
           'src/service/url-replacements-impl.js',
+      'extensions/amp-video/0.1/amp-video.js->' +
+          'src/service/video-manager-impl.js',
+      'extensions/amp-youtube/0.1/amp-youtube.js->' +
+          'src/service/video-manager-impl.js',
```

Comment:
amp-ooyala-player.js (merged recently) also uses video-manager and needs to be listed here.